src/kubernetes/server/server-deployment.yaml

```diff
@@ -40,6 +40,8 @@ spec:
           value: 'true'
         - name: SPRING_CLOUD_DATAFLOW_FEATURES_SCHEDULES_ENABLED
           value: 'true'
+        - name: SPRING_CLOUD_DATAFLOW_TASK_SCHEDULER_TASK_LAUNCHER_URL
+          value: 'docker://springcloud/spring-cloud-dataflow-scheduler-task-launcher:latest'
```

Comment:
Should be `2.3.0.BUILD-SNAPSHOT` for now as we currently need to manually replace all other image versions during a release.

Reply:
Ah yeah, we need to update CI to push out this image.

Reply:
Would this be `@project.version@` as well?